Online/Matchmaking/MatchmakingManager.cs

```diff
@@ -56,15 +62,25 @@ public static void InitLobbyManager()
             supported_matchmakers.Clear();
             instances.Clear();
 
-            if (OnlineManager.netIO is SteamNetIO) {
+            if (NetIOPlatform.isLANAvailable) {
+                supported_matchmakers.Add(MatchMakingDomain.LAN);
+                instances.Add(MatchMakingDomain.LAN, new LANMatchmakingManager());
+                currentDomain = MatchMakingDomain.LAN;
+            }
+
+            if (NetIOPlatform.isRouterAvailable) {
+                supported_matchmakers.Add(MatchMakingDomain.Router);
+                instances.Add(MatchMakingDomain.Router, new RouterMatchmakingManager());
+                currentDomain = MatchMakingDomain.Router;
+            }
+
+            if (NetIOPlatform.isSteamAvailable) {
                 instances.Add(MatchMakingDomain.Steam, new SteamMatchmakingManager());
                 supported_matchmakers.Add(MatchMakingDomain.Steam);
+                currentDomain = MatchMakingDomain.Steam;
             }
```

Comment:
```suggestion

            if (NetIOPlatform.isLANAvailable) {
                supported_matchmakers.Add(MatchMakingDomain.LAN);
                instances.Add(MatchMakingDomain.LAN, new LANMatchmakingManager());
                currentDomain = MatchMakingDomain.LAN;
            }


            if (NetIOPlatform.isSteamAvailable) {
instances.Add(MatchMakingDomain.Steam, new SteamMatchmakingManager());
supported_matchmakers.Add(MatchMakingDomain.Steam);
                currentDomain = MatchMakingDomain.Steam;
            }

            if (NetIOPlatform.isRouterAvailable) {
                supported_matchmakers.Add(MatchMakingDomain.Router);
                instances.Add(MatchMakingDomain.Router, new RouterMatchmakingManager());
                currentDomain = MatchMakingDomain.Router;
            }
```

This depends on whether we want to switch router to be preferred over Steam multiplayer. Since Router multiplayer is cross platform with Steam & Gog players.